src/overlays/actors/ovl_En_Neo_Reeba/z_en_neo_reeba.h

```diff
@@ -3,15 +3,31 @@
 
 #include "global.h"
 
+#define EN_NEO_REEBA_IS_LARGE(thisx) ((thisx)->params & 0x8000)
+
 struct EnNeoReeba;
 
 typedef void (*EnNeoReebaActionFunc)(struct EnNeoReeba*, GlobalContext*);
 
 typedef struct EnNeoReeba {
     /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x168];
+    /* 0x0144 */ SkelAnime skelAnime;
+    /* 0x0188 */ ColliderCylinder collider;
+    /* 0x01d4 */ Vec3s jointTable[18];
```

Comment:
```suggestion
    /* 0x01D4 */ Vec3s jointTable[18];
```
Note you could also delete the leading 0's in all these, but not a big deal.